core/controllers/feedback_test.py

```diff
@@ -462,14 +465,12 @@ def test_create_and_fetch_suggestions(self):
         self.assertEqual(len(threads), 6)
 
         # Get a suggestion.
-        full_thread_id = threads[0]['full_thread_id']
-        thread_id = feedback_services.get_thread_id_from_full_thread_id(
-            full_thread_id)
+        thread_id = threads[0]['thread_id']
         response_dict = self.get_json(
             '%s/%s/%s' % (feconf.FEEDBACK_THREAD_URL_PREFIX, self.EXP_ID,
                           thread_id))
-        self.assertEqual(
-            response_dict['suggestion']['id'], full_thread_id)
+        self.assertEqual(feedback_services.get_thread_id_from_full_thread_id(
```

Comment:
Shouldn't the response dict that's being sent to the frontend just use the thread id (and not the concatenation of the exp and thread ids)?